lib/vtk/commands/socks/setup.rb

```diff
@@ -0,0 +1,297 @@
+# frozen_string_literal: true
+
+require_relative '../../command'
+require 'tty-prompt'
+require 'fileutils'
+require 'erb'
+
+module Vtk
+  module Commands
+    class Socks
+      # Sets up socks access to the VA network
+      class Setup < Vtk::Command
```

Comment:
This class is too long but we haven't broken any subcommand classes up yet. I didn't want to think through how we should org this type of code separation. If you have suggestions, I'm all 👂👂👂.